src/test/java/uk/gov/justice/digital/delius/integration/secure/CustodyUpdateAPITest.java

```diff
@@ -99,11 +102,22 @@ public void updatePrisonInstitution() throws JsonProcessingException {
         assertThat(custody.getInstitution().getNomsPrisonInstitutionCode()).isEqualTo("MDI");
         verify(telemetryClient).trackEvent(eq("P2PTransferPrisonUpdated"), any(), isNull());
 
+        //custody record should have been updated
         final var custodyRecord = jdbcTemplate.query(
                 "SELECT * from CUSTODY where PRISONER_NUMBER = ?",
                 List.of(PRISON_BOOKING_NUMBER).toArray(),
                 new ColumnMapRowMapper()).get(0);
         assertThat(toLocalDateTime(custodyRecord.get("LAST_UPDATED_DATETIME"))).isCloseTo(LocalDateTime.now(), within(10, ChronoUnit.SECONDS));
+        assertThat(toLocalDateTime(custodyRecord.get("LOCATION_CHANGE_DATE")).toLocalDate()).isEqualTo(LocalDate.now());
```

Comment:
we have no end points for these entities so use raw SQL to test our side effects are correct

Reply:
Which is fine, but if you wanted an alternative you could use a SpyBean for the repository.